pkg/pillar/scripts/veth.sh

```diff
@@ -5,11 +5,18 @@ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 export PATH
 
 try() {
-   if ! "$@"; then
-      ip link del "$VIF_CTR" || :
-      ip link del "$VIF_NAME" || :
-      exit 1
-   fi
+    RETRIES=3
+    while [ "$RETRIES" -gt 0 ]; do
+        if ! "$@"; then
+            RETRIES="$(( RETRIES - 1 ))"
+            sleep 5
```

Comment:
I think 5 seconds is bit too long. What about we retry every second 15 times to avoid delaying boot of a native container too much?

Reply:
Btw. in pillar we retry with 1 second period: https://github.com/lf-edge/eve/blob/master/pkg/pillar/nireconciler/linuxitems/vlanbridge.go#L133-L169

Reply:
Yeah, makes sense... I will change it...